### PR TITLE
Feature: add focus to shell input when click anywhere

### PR DESCRIPTION
**Pull request by nakamuraos**
When click to shell, I want my mouse will focus to command input. This PR will add a function javascript to detect onclick (check not selecting text too).

### CODE REVIEW

File: shell.php [hunk 1]
```diff
@@ -427,6 +427,13 @@ function getQueryString() {
                 xhr.send(getQueryString());
             }
 
+            function focusShellCmd() {
+                var selection = window.getSelection();
+                if(!selection.toString()) {
+                    document.getElementById('shell-cmd').focus();
```
Comment (flozz) [inline shell.php:433]: No need to call `getElementById()` here, there is already a `eShellCmdInput` variable that contains this element

**Review by flozz — CHANGES_REQUESTED** [review]
Also, can you call your new function in the `onload` callback (replacing the `eShellCmdInput.focus();` line) : https://github.com/flozz/p0wny-shell/blob/master/shell.php#L434

**Comment by nakamuraos** [comment]
Thanks @flozz for review my code.

I'll close this PR since it has already in https://github.com/flozz/p0wny-shell/pull/23

File: shell.php (hunk 1 above)
Comment (nakamuraos) [reply shell.php:433]: No. The changes which you mention not correct. The code in this PR will execute every click, not only onload.

**Pull request closed without merging**